smarsy/parse.py

```diff
@@ -86,6 +86,15 @@ def get_headers():
     return open_json_file(file_path)
 
 
+def childs_page_return_right_login(response_page):
+    """
+    Receive HTML page from login function and check we've got expected source
+    """
+    smarsy_login = get_user_credentials()['username']
```

Comment:
username should be passed as a param then to avoid extra dependency (`get_user_credentials`).

Reply:
additionally there are few steps in the function already with single unit test. First step is get user credentials that is not tested. The only step that validates is tested

Reply:
def get_user_credentials() has no parameters, I call this function and then just take one key 'username'

Reply:
you need to pass username value to the `childs_page_return_right_login` to get rid of `get_user_credentials` dependency

Reply:
https://github.com/dkultasev/smarsy-py/pull/33/commits/272d698b809806c4db8eb9502037b77e3e770e03
Commit added without dependency